apps/etterlatte-saksbehandling-ui/client/src/components/behandling/beregningsgrunnlag/overstyrGrunnlagsBeregning/OverstyrBeregningGrunnlag.tsx

```diff
@@ -201,8 +201,9 @@ const OverstyrBeregningGrunnlag = (props: {
           <CalculatorIcon fontSize="1.5rem" aria-hidden />
           <Heading size="small">Beregningsgrunnlag for overstyrt beregning</Heading>
         </HStack>
-
-        <OverstyrtBeregningsgrunnlagTable />
+        <Box maxWidth="70rem">
```

Comment:
Har vi ikkje ei maksgrense på 42,5?

Reply:
Det er for ren tekst slik at tekst er mer leslig, for andre komponenter er det litt hipp som happ